dashboard/app/models/user.rb

```diff
@@ -246,6 +247,19 @@ def delete_course_as_facilitator(course)
     courses_as_facilitator.find_by(course: course).try(:destroy)
   end
 
+  def log_admin_save
+    # Do not log for development or adhoc environments.
+    return unless [:test, :staging, :levelbuilder, :production].include? rack_env
```

Comment:
I included `:test` so that the automated tests of this feature would invoke the logging mechanism (which is mock'd).  Is there a way to stub `ChatClient.message` for all invocations in the `:test` environment?